diff -Naur qrq-0.2.1/Makefile qrq-0.2.1-mod/Makefile
--- qrq-0.2.1/Makefile	2011-09-19 22:01:41.000000000 +0200
+++ qrq-0.2.1-mod/Makefile	2011-09-20 23:48:52.755694241 +0200
@@ -90,13 +90,13 @@
 	install -d -v                      $(DESTDIR)/share/qrq/
 	install -d -v                      $(DESTDIR)/share/man/man1/
 	install -d -v                      $(DESTDIR)/bin/
-	install -s -m 0755 qrq             $(DESTDIR)/bin/
-	install    -m 0755 qrqscore        $(DESTDIR)/bin/
-	install    -m 0644 qrq.1           $(DESTDIR)/share/man/man1/
-	install    -m 0644 callbase.qcb    $(DESTDIR)/share/qrq/
-	install    -m 0644 english.qcb     $(DESTDIR)/share/qrq/
-	install    -m 0644 qrqrc           $(DESTDIR)/share/qrq/
-	install    -m 0644 toplist         $(DESTDIR)/share/qrq/
+	install -p -m 0755 qrq             $(DESTDIR)/bin/
+	install -p -m 0755 qrqscore        $(DESTDIR)/bin/
+	install -p -m 0644 qrq.1           $(DESTDIR)/share/man/man1/
+	install -p -m 0644 callbase.qcb    $(DESTDIR)/share/qrq/
+	install -p -m 0644 english.qcb     $(DESTDIR)/share/qrq/
+	install -p -m 0644 qrqrc           $(DESTDIR)/share/qrq/
+	install -p -m 0644 toplist         $(DESTDIR)/share/qrq/
 
 	
 uninstall:
